documentation/src/main/asciidoc/server_guide/server_guide.adoc

```diff
@@ -607,6 +607,45 @@ The REST connector differs from the above connectors because it piggybacks on th
 
 ----
 
+==== Multi-tenant router
```

Comment:
There are multiple things configured at endpoint level such as security (SSL and SASL), socket bindings , topology state transfer etc. If we decided to simply implement multiple Cache Containers per endpoint, all tenants would need to share the same settings (which is probably something we don't want to do).

Router does not touch those settings, it only create a facade for transport. I think it's more flexible approach.